extensions/wikia/Blogs/css/oasis.scss

```diff
@@ -261,7 +261,7 @@
 	margin: 4px 2px 0;
 }
 
-wpCategoryInput {
+#wpCategoryInput {
```

Comment:
Neither do I, but it was a bug in the CSS file - and based on https://github.com/Wikia/app/pull/7365#discussion-diff-31339417R259 I've fixed declaration. I'm not sure that we use it, but it may be used by our users.